### ISSUE TITLE
Ability to use VBA-M core for GB/GBC instead of Gambette

### ISSUE THREAD

**Issue by upintheairsheep** [rileytestut/Delta#309]
Can you add the ability to replace the Gambette code with VBA-M core used for GBA?

**Comment by haoranliu666** [comment]
Isn't Delta using VBA-M now according to this GitHub page?

**Comment by Papermanzero** [comment]
mGBA would be the better option for GB GBC and GBA

**Comment by meechgalhuquot** [comment]
> mGBA would be the better option for GB GBC and GBA

Agreed. The "Ignited" emulator that is a fork of Delta already switched to this core months ago for a reason.
